fuzzy_file_nav.py

```diff
@@ -600,7 +600,7 @@ def run(self):
                     os.remove(full_name)
             except Exception:
                 errors = True
-                error("Error deleting %d!" % full_name)
+                error("Error deleting %s!" % full_name)
 
```

Comment:
This seemed wrong to me, I got an error when testing the new commands because of %d so I changed here too.